### PR TITLE
[config] Add config JSON generator

### PR DESCRIPTION
**Pull request by jaylees14**
Add a generator for `config.json` from spec-golang.

Nothing too exciting - will do the mapping after because of all tests that will need updating

### CODE REVIEW

File: src/main/scala/temple/generate/server/config/ServerConfigGenerator.scala [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import io.circe.syntax._
+import temple.ast.Metadata.Database
+import temple.ast.Templefile.Ports
+import temple.generate.server.config.ast.{PostgresConfig, ServerConfig}
+
+object ServerConfigGenerator {
+
+  def generate(serviceName: String, database: Database, services: Map[String, String], ports: Ports): String = {
+    val databaseConfig = database match {
+      case Database.Postgres => PostgresConfig(serviceName + "-db")
```
Comment (jaylees14) [inline src/main/scala/temple/generate/server/config/ServerConfigGenerator.scala:12]: @Lewiky this is what you do in Kube right?

**Review by c-harding — APPROVED** [review]
nice and simple

File: src/main/scala/temple/generate/server/config/ServerConfigGenerator.scala (hunk 1 above)
Comment (Lewiky) [reply src/main/scala/temple/generate/server/config/ServerConfigGenerator.scala:12]: Yeah bruh

**Review by Lewiky — APPROVED** [review]
Nice

**Pull request merged**